src/main/resources/db/migration/V202409231350__create_feedback_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE feedback
+(
+    id           BIGSERIAL PRIMARY KEY,
+    employee_id  BIGINT,
+    content      VARCHAR(225) NOT NULL,
```

Comment:
make this a text type
```suggestion
    content      text NOT NULL,
```